challenge-13/submissions/onomica/solution-template.go

```
@@ -0,0 +1,174 @@
+
+	return &p, nil
+}
+
+// UpdateProduct updates an existing product
+func (ps *ProductStore) UpdateProduct(product *Product) error {
+	if _, err := ps.GetProduct(product.ID); err != nil {
+		return err
+	}
+
+	query := `UPDATE products SET name = ?, price = ?, quantity = ?, category = ? WHERE id = ?`
+	_, err := ps.db.Exec(query, product.Name, product.Price, product.Quantity, product.Category, product.ID)
```

golangci-lint (2.12.2)

[error] (*database/sql.DB).Exec must not be called. use (*database/sql.DB).ExecContext

(noctx)